.circleci/config/jobs/website-docker-image.yml

```diff
@@ -0,0 +1,21 @@
+docker:
+  - image: circleci/buildpack-deps
+steps:
```

Comment:
I would recommend adding the line `shell: /usr/bin/env bash -euo pipefail -c` just before `steps:` to make sure these scripts fail appropriately, all the executors defined in `.circleci/config/@config.yml` already define this shell, so this would be consistent with those.

Reply:
Thanks for the feedback here, just added that line as suggested 🌟